### DISCUSSION TITLE
Tailwind CSS has slowed down my blog by 10 points on Pagespeed Mobile

### DISCUSSION THREAD

**Discussion by nsursock** [tailwindlabs/tailwindcss#1927] [Help]
Hello,

I like tailwind CSS a lot and I want to use it on my blog. But some pages display an error when using the library. To be more precise, I'm talking about Largest Contentful Paint.

When I don't get an error, I'm losing 10 points on Google Pagespeed for mobile which means Tailwind made my website slower by 0.5 seconds. My goal is to be fast and anything that makes my site slower can't be included. 

Please have a look:
 - Error: https://soundsirius.digitalpress.blog/products/
 - Slow: https://soundsirius.digitalpress.blog/ (tailwind part is in the footer, it's a subscribe widget)

Many thanks

**Comment by royvanv** [comment]
You are using the full Tailwind CSS from the CDN, instead of a custom build.

These 2 links should help you to optimize your CSS file size:
https://tailwindcss.com/docs/installation/
https://tailwindcss.com/docs/controlling-file-size

**Comment by estevanmaito** [comment]
Did you try this? https://github.com/tailwindcss/tailwindcss/discussions/1912

**Comment by nsursock** [comment]
Sadly I don't have much time: I'm working alone on my music blog. I need a simple copy/paste solution with only a few tweaks.

**Comment by nsursock** [comment]
Can you tell me why my subscribe widget in the footer built with tailwind generates so much space in the bottom? What if I need to add another widget? I need to control this space. So far, I had no luck.

**Reply by royvanv** [reply]
Because you used the class `h-screen` 😉

**Reply by nsursock** [reply]
great many thanks !!!

**Comment by nsursock** [comment]
Another thing: on my testimonials page, I'd like to have some horizontal space between testimonials. How do I do that? Sometimes it works, sometimes it doesn't. https://soundsirius.digitalpress.blog/testimonials/
